packages/react/src/components/OptionsMenu/OptionsMenuList.test.tsx

```diff
@@ -0,0 +1,307 @@
+      <li>option 1</li>
+      <li>option 2</li>
+    </OptionsMenuList>
+  );
+
+  fireEvent.keyDown(screen.getAllByRole('menuitem')[0], {
+    key: 'Tab',
+    keyCode: tab,
+    which: tab
+  });
+
+  expect(defaultProps.onClose).toBeCalled();
+});
+
+test('should call onClose when clicked outside', () => {
+  render(
+    <OptionsMenuList {...defaultProps} show={true}>
+      <li>option 1</li>
+      <li>option 2</li>
+    </OptionsMenuList>
+  );
+
+  fireEvent.click(document.body);
+
+  expect(defaultProps.onClose).toBeCalled();
+});
+
+test('should fire onSelect when menu item is clicked', () => {
+  const onSelect = jest.fn();
+
+  render(
+    <OptionsMenuList {...defaultProps} onSelect={onSelect}>
+      <li>option 1</li>
+      <li>option 2</li>
+    </OptionsMenuList>
+  );
+
+  fireEvent.click(screen.getAllByRole('menuitem')[0]);
+
+  expect(onSelect).toBeCalled();
+});
+
+test('should fire onSelect when menu item is selected with space', () => {
+  const onSelect = jest.fn();
+
+  render(
+    <OptionsMenuList {...defaultProps} onSelect={onSelect}>
+      <li>option 1</li>
+      <li>option 2</li>
+    </OptionsMenuList>
+  );
+
+  fireEvent.keyDown(screen.getAllByRole('menuitem')[0], {
+    key: ' ',
+    keyCode: space,
+    which: space
+  });
+
+  expect(onSelect).toBeCalled();
+});
+
+test('should fire onSelect when menu item is selected with enter', () => {
+  const onSelect = jest.fn();
+
+  render(
+    <OptionsMenuList {...defaultProps} onSelect={onSelect}>
+      <li>option 1</li>
+      <li>option 2</li>
+    </OptionsMenuList>
+  );
+
+  fireEvent.keyDown(screen.getAllByRole('menuitem')[0], {
+    key: 'Enter',
+    keyCode: enter,
+    which: enter
+  });
+
+  expect(onSelect).toBeCalled();
+});
+
+test('should fire onClose when menu item is selected', () => {
+  render(
+    <OptionsMenuList {...defaultProps}>
+      <li>option 1</li>
+      <li>option 2</li>
+    </OptionsMenuList>
+  );
+
+  fireEvent.click(screen.getAllByRole('menuitem')[0]);
+
+  expect(defaultProps.onClose).toBeCalled();
+});
+
+test('should not fire onClose when menu item is selected and default prevented', () => {
+  const event = new MouseEvent('click', {
+    bubbles: true,
+    cancelable: true
+  });
+
+  event.preventDefault();
+
+  render(
+    <OptionsMenuList {...defaultProps}>
+      <li>option 1</li>
+      <li>option 2</li>
+    </OptionsMenuList>
+  );
+
+  screen.getAllByRole('menuitem')[0].dispatchEvent(event);
+
+  expect(defaultProps.onClose).not.toBeCalled();
+});
+
+test('should click child links with click events', () => {
+  const onClick = jest.fn();
+
+  render(
+    <OptionsMenuList {...defaultProps}>
+      <li>
+        <a href="#foo" onClick={onClick}>
+          Click me!
+        </a>
+      </li>
+      <li>option 2</li>
+    </OptionsMenuList>
+  );
+
+  fireEvent.click(screen.getByRole('link'));
+
+  expect(onClick).toBeCalledTimes(1);
+});
+
+test('should focus child links with keydown events', () => {
```

Comment:
This test name mentions focus, but it looks like we're checking for `click` here. Is this test right?